### PR TITLE
gpsim 0.31.0

### PR DESCRIPTION
**Pull request by alebcay**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by Moisan** [comment]
```
modules.cc:50:10: fatal error: 'error.h' file not found
#include "error.h"
         ^~~~~~~~~
```

**Comment by alebcay** [comment]
Builds/CI are passing now.

File: Formula/gpsim.rb [hunk 1]
```diff
@@ -15,10 +14,16 @@ class Gpsim < Formula
...
   depends_on "pkg-config" => :build
+  depends_on "gettext"
   depends_on "glib"
   depends_on "popt"
+  depends_on "readline"
 
   def install
+    ENV.cxx11
+
+    inreplace "src/modules.cc", "error.h", "errors.h"
```
Comment (Moisan) [inline Formula/gpsim.rb]: Is upstream aware of this issue?

Comment (alebcay) [reply Formula/gpsim.rb]: Emailed the mailing `gpsim-devel` mailing list and also opened a bug ticket on SourceForge: https://sourceforge.net/p/gpsim/bugs/245/.

**Pull request closed without merging**